flixel/addons/effects/chainable/FlxWaveEffect.hx

```diff
@@ -71,13 +75,14 @@ class FlxWaveEffect implements IFlxEffect
 	 * @param	Wavelength	How long waves are.
 	 * @param	Direction	Which Direction you want the effect to be applied (HORIZONTAL or VERTICAL).
 	 */
-	public function new(?Mode:FlxWaveMode, Strength:Int = 10, Center:Float = -1, Speed:Float = 3, Wavelength:Int = 5, ?Direction:FlxWaveDirection)
+	public function new(?Mode:FlxWaveMode, Strength:Int = 10, Center:Float = -1, Speed:Float = 3, Wavelength:Int = 5, ?Direction:FlxWaveDirection, ?Interlaced:Bool = false)
```

Comment:
No, `?` is fine for non-basic types (for example the enum arguments).

Changing the sorting would break backwards compatibility.